### PR TITLE
manifest: update zephyr

### PR DESCRIPTION
**Pull request by ioannisg**
Update Zephyr to point to the result of this PR:

https://github.com/NordicPlayground/fw-nrfconnect-zephyr/pull/166

Additional manifest updates for MBEDTLS and MCUMGR

Signed-off-by: Ioannis Glaropoulos <Ioannis.Glaropoulos@nordicsemi.no>

### CODE REVIEW

File: west.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ manifest:
     - name: fw-nrfconnect-zephyr
       path: zephyr
       west-commands: scripts/west-commands.yml
-      revision: 7e64ac737daead12d2dbdda5daec04188243a81c
+      revision: f69d8d61a934533de2dfc5c3ca915b82910acd8d
```
Comment (mbolivar) [inline west.yml]: why a sha and not `pull/166/head` while the other PR is still in review? the final sha will be different depending on who commits it.

Comment (rlubos) [reply west.yml]: Indeed you should use `pull/166/head`. The PR branch lies on your fork which is a different remote - `west` cannot find it.

Comment (ioannisg) [reply west.yml]: Right, fixed that now. @rlubos @mbolivar

Comment (thst-nordic) [reply west.yml]: pull/166/head is a 'branch' name that can move (not good for traceability) .  I think the policy is west  manifests should have sha's or tag's( which don't move)
@carlescufi   input?

**Comment by rlubos** [comment]
@ioannisg Quite a few samples does not build in the CI:
https://jenkins-ncs.nordicsemi.no/blue/organizations/jenkins/fw-nrfconnect-nrf/detail/PR-883/8/pipeline

**Comment by ioannisg** [comment]
@carlescufi @mbolivar @rlubos could you please review the additional updates of the manifest?
- the mbedtls revision
- the mcumgr revision

File: west.yml [hunk 2]
```diff
@@ -69,9 +69,9 @@ manifest:
       path: test/cmock/vendor/unity
       revision: 031f3bbe45f8adf504ca3d13e6f093869920b091
       remote: throwtheswitch
-    - name: mbedtls
-      revision: mbedtls-2.13.1
```
Comment (ioannisg) [inline west.yml]: @carlescufi @marti I am not sure, anymore if this is correct: are we, now moving mbedtls into modules?
If so, then we need to set the path to modules/lib/mbedtls, correct?
And also; will this be sufficient for west to checkout/update mbedtls? I.e. do we need anything else?

Comment (ioannisg) [reply west.yml]: @tejlmand ^^

Comment (rlubos) [reply west.yml]: Didn't we talk recently about keeping two mbedTLS instances for now? One for the downstream, just as it was, and one for the upstream as a replacement for the removed `ext` copy? The latter one respecting the upstream path for the mbedTLS, i. e. `modules/crypto/mbedtls`.

Comment (tejlmand) [reply west.yml]: No, this version should be kept.
For time being we still need both, which were also the case before the split of zephyr mbedtls upstream.

**Comment by rlubos** [comment]
@pdunaj nrf_desktop tests failed, could you take a look?

**Comment by pdunaj** [comment]
Hi @rlubos , sure. It seems that mouse connected but at some point communication stopped. I don't see anything in the logs that would indicate a problem.
Hi @wiminordic , maybe you will see more?

I can check it locally. It seems that FW crashed or something but strange that there are no logs about it.

**Comment by pdunaj** [comment]
@rlubos , I checked this PR locally and it seems that bluetooth does not work correctly. 
I am able to connect to my Win7 but I am unable to properly connect to Android. If I would have to bet on something I would say that either privacy or non-legacy connections are broken.
Current master works fine.

**Comment by rlubos** [comment]
@pdunaj I did a follow-up upmerge which introduced some BLE fixes from upstream. The nrf_desktop CI seems fixed now. I'd appreciate if you could verify locally if it works for you as well.

**Comment by mbolivar** [comment]
> @carlescufi @mbolivar @rlubos could you please review the additional updates of the manifest?

Other than the remaining PRs which need to be merged, the west.yml changes look mergeable to me, as soon as CI and other testing passes.

**Comment by pdunaj** [comment]
Hi @rlubos , thanks. Mouse seems to be working fine now on the Android.

**Comment by pdunaj** [comment]
Hi @wiminordic , could you trigger our full CI against this PR?

**Comment by pdunaj** [comment]
@rlubos , by default nrf_desktop CI is not run for every PR but only the ones that have nrf_desktop mentioned in the title. It would be good to run complete CI during every upmerge. Maybe @wiminordic could somehow detect that upmerge is in progress and start CI for it (e.g. for every PR modifying the manifest file)?

**Comment by wiminordic** [comment]
> Hi @wiminordic , could you trigger our full CI against this PR?

It should run our full CI for the next commit on this PR.

**Comment by rlubos** [comment]
@pdunaj I'm fine with that, though I'm not the one to decide. Optionally, we can consider adding a keyword to the upmerge PRs which you could detect - not all manifest updates are significant for nrf_desktop.

**Comment by rlubos** [comment]
@wiminordic There will be no more commits/pushes here, will simple re-run be enough?

**Comment by wiminordic** [comment]
All further commits/pushes here should trigger extended CI.

**Comment by pdunaj** [comment]
Hi @wiminordic , thanks.

Hi @rlubos , `nrf_desktop` uses zephyr, nrfxlib and mcuboot.  These are the most frequently updated repos I believe. Also our full CI is not really taking that much time plus it is not required for PR to be mergable (merely informative). I think it would benefit us to have this check. @carlescufi , what do you think?

**Comment by rlubos** [comment]
All dependencies merged, now waiting for the final CI round.

**Comment by carlescufi** [comment]
> Hi @wiminordic , thanks.
> 
> Hi @rlubos , `nrf_desktop` uses zephyr, nrfxlib and mcuboot. These are the most frequently updated repos I believe. Also our full CI is not really taking that much time plus it is not required for PR to be mergable (merely informative). I think it would benefit us to have this check. @carlescufi , what do you think?

I am not sure which check you are referring to @pdunaj, can you elaborate?

**Comment by pdunaj** [comment]
Hi @carlescufi , sure. 
CI run against nrf_desktop tagged PR have more tests executed. e.g. https://jenkins-ncs.nordicsemi.no/blue/organizations/jenkins/desktop52_verification/detail/master/3102/tests
compared to 'normal' PR e.g.
https://jenkins-ncs.nordicsemi.no/blue/organizations/jenkins/desktop52_verification/detail/master/3073/tests

I see that at the moment both cases are hooked to GitHub using the same name `nrf_desktop_test` which is not required. When @wiminordic extends full nrf_desktop CI to also run on manifest updating commits, more tests will be executed.

**Comment by carlescufi** [comment]
@pdunaj got you, agree with you.

**Comment by wiminordic** [comment]
@carlescufi continuous-integration/jenkins/nrf_desktop_test
It can have varied scope. From now on it will run more extensive test suite of nrf_desktop tests for PRs containing "manifest" in the PR title. It takes around 20min to complete. If it is not ok, it can be reverted to previous smoke-test scope which takes less than 2min. We can talk outside GH if you want more details.

**Pull request merged**